probe-rs/src/architecture/arm/sequences.rs

```diff
@@ -964,6 +965,55 @@ pub trait ArmDebugSequence: Send + Sync + Debug {
         Ok(())
     }
 
+    /// This ARM sequence is called if an image was flashed to RAM directly.
+    /// It will perform the necessary preparation to run that image.
+    fn ram_flash_start(
```

Comment:
I would name this differently. This thing does nothing with flash/flashing. It's preparation fro running from flash.